u8timeseries/tests/test_metrics.py

```diff
@@ -17,6 +18,10 @@ class MetricsTestCase(unittest.TestCase):
     series2: TimeSeries = TimeSeries(pd_series2)
     series3: TimeSeries = TimeSeries(pd_series3)
 
+    @classmethod
+    def setUpClass(cls):
+        logging.disable(logging.CRITICAL)
```

Comment:
what is this doing?

Reply:
This suppresses the logging of all log messages below the level of CRITICAL. In this case, this happens before the execution of all test cases for timeseries. I chose this approach here because for test_timeseries.py, we do not need any log messages.